src/api/users.js

```diff
@@ -66,6 +64,12 @@ export async function authenticate(ctx, email) {
   }
 }
 
+export const logout =  async function(ctx) {
+  ctx.session = null;
```

Comment:
Does passport automatically delete the session document from the DB ?

Reply:
nope, I need to do that manually, I will address this